if_any() and if_all() to use inside filter() (#5660)
tidyverse/dplyr: vignettes/colwise.Rmd

`across()` doesn't work with `select()` or `rename()` because they already use tidy select syntax; if you want to transform column names with a function, you can use  `rename_with()`.

### filter()

We cannot directly use `across()` in `filter()` because we need an extra step to combine 
the results. To that end, `filter()` has two special purpose companion functions: 

> * `if_any()` to keep the rows for which the predicate is true for *at least one* selected

Reviewer: ```suggestion
* `if_any()` keeps the rows where the predicate is true for *at least one* selected
```

  column: 
  
```{r}
starwars %>% 
  filter(if_any(everything(), ~ !is.na(.x)))
```
  

> * `if_all()` to keep the rows where the predicate is true for *all* selected columns:

Reviewer: ```suggestion
* `if_all()` keeps the rows where the predicate is true for *all* selected columns:
```
